examples/MCSH_potential/C_totaldensity_4.g

```diff
@@ -0,0 +1,4 @@
+8.635841277873077e-06	0.0005060277354303857
+183.6211672291742	75.90796797063469
+358.18820452947995	447.3395875309654
+209.57398363206482	4531.367326575614
```

Comment:
Can we find a different way to include these, don't think we should have it in the `examples` folder. Where do these come from exactly? Can we get this for atomic numbers 1-70 or so and we can just have that be part of the code to be looked up.

Reply:
These are like the pseudo potential files for DFT calculations. it's something needed to run the calculations, but also something up for optimization. I created these ones for the model example, we are planning to create these files for all the elements, but it would take a long time

Reply:
Is it possible to include a script to generate these? 

It takes a long time for the calculations themselves, or just a lot of time optimizing the parameters?

Reply:
These are the files that defines the gaussians used to represent one element type. I'm also experimenting the with the strategies to generate these files, now I'm just using the psp files used in our group and fit gaussians to it, but it's totally experimental at this point, so might not be a good idea to share my duck-taped code. maybe I could share the ones I have created and comfortable with, including H,C,O,N,F,Fe,Cu, and possibly Pt soon

Reply:
Even a duck-taped script using GBRV pseudopotentials would be helpful I think. It can always be cleaned up after, but would be super helpful to see where these are coming from.

Reply:
sounds good, I'll clean up the code a bit and include in this PR